src/controller/insertcontent.js

```diff
@@ -226,6 +226,12 @@ class Insertion {
 		if ( node.is( 'element' ) ) {
 			this.handleNodes( node.getChildren(), context );
 		}
+		// When disallowed node is a text but text is allowed in current position it means that our node
+		// contains disallowed attributes and we have to remove them.
+		else if ( node.is( 'text' ) && this.schema.check( { name: '$text', inside: this.position } ) ) {
```

Comment:
Is `node.is( 'text' )` needed here? We are in `else` after `node.is( 'element' )` so AFAIR the node can be only text node.